examples/rocket/Dockerfile

```diff
@@ -8,6 +8,4 @@ ENV ROCKET_ENV=staging
 WORKDIR /app
 COPY . .
 
-RUN cargo build --release
-
-CMD ROCKET_PORT=$PORT ./target/release/rocket
+CMD ROCKET_PORT=$PORT cargo run --release
```

Comment:
This change moves the Rust compilation from the Docker build step to the Docker run step, which isn't great for the Railway deployment pipeline. We want all artifacts to be inside the built Docker image to minimize container start-up time.